Vizzuality/heco-invest
# Let 777 fe allow the user to change their password

This PR adds the change password page

## Acceptance criteria

On the user settings page, the user is able to change their password

The form asks the user:

- Their current password
- Their new password
- A confirmation of their new password

Design: [HeCo - Work in progress](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=3465%3A52408) 

Note: the user settings are accessed from the “My preferences” item of the avatar menu ([HeCo - Work in progress](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A5482))

## Testing instructions

1. Login and click on 'menu' - 'my preferences'. You should go to 'settings/information'
2. Click on the tab 'password'. You should go to 'settings/password' and see the change password form
3. Fill the form
  - If you fill a field incorrectly, it should display an error below the field
<img width="769" alt="Screenshot 2022-07-29 at 15 48 16" src="https://user-images.githubusercontent.com/48164343/181774264-b42c8a27-da54-4da2-8de6-d11ca7244811.png">

 4. Submit the form clicking on the 'update password' button
  - If you try to submit the form incomplete or with errors, error messages should be shown below the fields
  - If you input your current password incorrectly, an API error should be shown above the button and the form should be reseted. 
<img width="769" alt="Screenshot 2022-07-29 at 15 47 43" src="https://user-images.githubusercontent.com/48164343/181774596-de96c54b-535c-41b2-896d-4edea9b588c1.png">
  - If the action is successful, you should see a success message and the form should be reseted. 
<img width="769" alt="Screenshot 2022-07-29 at 15 48 06" src="https://user-images.githubusercontent.com/48164343/181774313-15543e47-531f-409a-87ec-8953d77137e0.png">
## Tracking

[LET-777](https://vizzuality.atlassian.net/browse/LET-777)
